content/en/docs/tasks/traffic-management/ingress/secure-ingress-sds/index.md

```diff
@@ -539,6 +539,20 @@ $ kubectl create -n istio-system secret generic httpbin-credential  \
 
     {{< /text >}}
 
+1. Instead of creating a single secret `httpbin-credential` that holds all the credentials, you can
+   also create two separate secrets, `httpbin-credential` and `httpbin-credential-cacert`.
+   `httpbin-credential` holds server key and server certificate. `httpbin-credential-cacert` holds
+   the client CA certificate, and it should contain `-cacert` suffix.
+
+    {{< text bash >}}
```

Comment:
Please introduce the code block to readers. This provides them with the context they need to link the commands to the text. 
```suggestion
Create the two separate secrets with the following commands:

    {{< text bash >}}
```